src/kernels/xform_bidirect_winograd_code.inc

```diff
@@ -91,6 +91,7 @@
 
 default pipe_config, 0
 default use_exec_for_vmem, 1 // todo test and remove
+default XDLOPS_CONV_GEMM, 0
```

Comment:
I suggest to name the variable by it's meaning. What it really do is switching layout. Please name it accordingly.